EasyPost/Client.cs

```diff
@@ -1,91 +1,89 @@
 ﻿using Newtonsoft.Json;
```

Comment:
You need to rebase so the user-agent changes aren't included in this PR.

Reply:
Patched in bc436ee